### PR TITLE
fix: Cypress failure on advance search and redirection tests

### PR DESCRIPTION
**Pull request by ShaileshParmar11**
### Describe your changes :

I worked on the ..... because ...

#
### Type of change :

- [x] Bug fix
- [x] Improvement
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/cypress/constants/redirections.constants.js [hunk 1]
```diff
@@ -14,26 +14,26 @@
...
-  },
-  pipelines: {
-    testid: '[data-testid="pipelines"]',
-    url: `${BASE_URL}/explore/pipelines`,
-  },
-  mlmodels: {
-    testid: '[data-testid="mlmodels"]',
-    url: `${BASE_URL}/explore/mlmodels`,
-  },
+  // tables: {
```
Comment (aniketkatkar97) [inline openmetadata-ui/src/main/resources/ui/cypress/constants/redirections.constants.js:17]: Can we please add the reason for commenting these parts?

File: openmetadata-ui/src/main/resources/ui/cypress/e2e/Flow/AdvanceSearch.spec.js [hunk 2]
```diff
@@ -31,15 +31,13 @@ import {
     verifyResponseStatusCode
 } from '../../common/common';
 
-import { API_SERVICE } from '../../constants/constants';
 import { MYSQL } from '../../constants/service.constants';
 
 const service_name = MYSQL.serviceName;
 
-describe('Advance search should work properly for all fields', () => {
+describe.skip('Advance search should work properly for all fields', () => {
```
Comment (aniketkatkar97) [inline openmetadata-ui/src/main/resources/ui/cypress/e2e/Flow/AdvanceSearch.spec.js:38]: Is this intentional? if yes can we add the reason as well?

File: openmetadata-ui/src/main/resources/ui/cypress/e2e/Flow/AdvanceSearch.spec.js [hunk 3]
```diff
@@ -116,7 +114,7 @@ describe('Advance search should work properly for all fields', () => {
   });
 });
 
-describe('Advance search should work properly for Add Group functionality', () => {
+describe.skip('Advance search should work properly for Add Group functionality', () => {
```
Comment (aniketkatkar97) [inline openmetadata-ui/src/main/resources/ui/cypress/e2e/Flow/AdvanceSearch.spec.js:117]: Same, can we add comments with the reason behind skipping?

**Review by Sachin-chaurasiya — DISMISSED** [review]

**Review by aniketkatkar97 — COMMENTED** [review]
left some comments

File: openmetadata-ui/src/main/resources/ui/cypress/e2e/Flow/AdvanceSearch.spec.js (hunk 3 above)
Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/cypress/e2e/Flow/AdvanceSearch.spec.js:117]: Thanks for pointing this out but its already discussed, for time being we need to skip this test due to the ES issue

**Review by aniketkatkar97 — APPROVED** [review]

**Comment by ShaileshParmar11** [comment]
Closing this the PR as ES issue already fixed in main

**Pull request closed without merging**